### PR TITLE
Clarify rules for packing and unpacking in Section 8.1

### PR DESCRIPTION
**Pull request by JonathanGregory**
See issue #374 for discussion of these changes.

# Release checklist
- [NA] Authors updated in `cf-conventions.adoc`?
- [NA] Next version in `cf-conventions.adoc` up to date? Versioning inspired by [SemVer](https://semver.org).
- [Y] `history.adoc` up to date?
- [Y] Conformance document up-to-date?

### CODE REVIEW

**Comment by mankoff** [comment]
I think this change is a great improvement to the documentation. Thank you.

File: ch08.adoc [hunk 1]
```diff
@@ -28,10 +28,10 @@ When scaled data are written, the application should first subtract the offset a
...
-If the **`scale_factor`** and **`add_offset`** attributes are of the same data type as the associated variable, the unpacked data is assumed to be of the same data type as the packed data.
-However, if the **`scale_factor`** and **`add_offset`** attributes are of a different data type from the variable (containing the packed data) then the unpacked data should match the type of these attributes, which must both be of type **`float`** or both be of type **`double`**.
-An additional restriction in this case is that the variable containing the packed data must be of type **`byte`**, **`short`** or **`int`**.
-It is not advised to unpack an **`int`** into a **`float`** as there is a potential precision loss.
+
+When packed data is written, the **`scale_factor`** and **`add_offset`** attributes must be of the same type as the unpacked data, which must be either **`float`** or **`double`**. Data of type **`float`** must be packed into one of these types: **`byte`**, **`unsigned byte`**, **`short`**, **`unsigned short`**. Data of type **`double`** must be backed into one of these types: **`byte`**, **`unsigned byte`**, **`short`**, **`unsigned short`**, **`int`**, **`unsigned int`**.
```
Comment (kmuehlbauer) [inline ch08.adoc:32]: Nit: backed -> packed

**Review by kmuehlbauer — COMMENTED** [review]
Thanks @JonathanGregory. I agree with @mankoff, this is a great improvement.

File: ch08.adoc (hunk 1 above)
Comment (JonathanGregory) [reply ch08.adoc]: Whoops. Now fixed. Thanks.

**Pull request merged**